src/concrete/ml/common/utils.py

```diff
@@ -668,3 +682,50 @@ def process_rounding_threshold_bits(rounding_threshold_bits):
         rounding_threshold_bits = {"n_bits": n_bits_rounding, "method": method}
 
     return rounding_threshold_bits
+
+
+def serialize_encrypted_values(
```

Comment:
Do we need these outside of deployment? If not I would keep these in the deployment directory.

Reply:
well, currently I'm using them in the notebook on the server side to serialize/deserialize values outside of the `FheModelServer.run`, but now that I'm thinking of it, it's not mandatory because they are not tuples. Instead, they can directly call concrete's method (`.serialize()` or `fhe.Value.deserialize(...)`) 

also I do something similar in pandas encrypted so I initially thought I could merge both, but not really actually

finally I did not want to make the `fhe_client_server.py` file heavier, but I could create a `utils.py` in `deployment` for that actually, let me know what you think !

Reply:
I moved them to a `_utils.py` file the deployment directory